tokio/src/sync/tests/loom_watch.rs

```diff
@@ -34,3 +35,29 @@ fn smoke() {
         th.join().unwrap();
     })
 }
+
+#[test]
+fn wait_for_test() {
+    loom::model(move || {
+        let (tx, mut rx) = watch::channel(false);
+
+        let tx_arc = Arc::new(tx);
+        let tx1 = tx_arc.clone();
+        let tx2 = tx_arc.clone();
+
+        let th1 = thread::spawn(move || {
+            for _ in 0..3 {
```

Comment:
This test takes quite a long time with `LOOM_MAX_PREEMPTIONS=2`, as configured in CI. This makes it run faster.
```suggestion
            for _ in 0..2 {
```